custom_components/healthchecksio/sensor.py

```
@@ -0,0 +1,95 @@
+    def __init__(self, hass, check_data, config_entry):
+        self.hass = hass
+        self.config_entry = config_entry
+        self.check_data = check_data
+        self._attr_name = None
+        self._attr_unique_id = self.check_data.get(ATTR_PING_URL, "").split("/")[-1]
+        self._attr_extra_state_attributes = {}
+        self._attr_native_value = None
+        self._attr_icon = ICON_DEFAULT
+        self.check = {}
+
+    async def async_update(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `async_update`

Add return type annotation: `None`

(ANN201)